scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh

```diff
@@ -56,13 +56,16 @@ gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=s
 # prod only needs the terra-billing group
 if [[ "${ENV}" == "prod" ]]; then
   gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@firecloud.org --role=roles/owner
+  gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@firecloud.org --role=roles/resourcemanager.projectMover
```

Comment:
just curious (maybe answered by the ticket) -- why does terra-billing need this permission? If it's to add projects to folders, could we use the CRM SA for that?

We updated all uses of CloudResourceManager to use the CloudResourceManager SA, so I just wanted to make sure we're using the right SAs for the right jobs. 
https://github.com/broadinstitute/rawls/blob/v2_workspaces/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala#L1276

Reply:
looks like the CRM SA is just the Rawls SA